jest.config.js

```diff
@@ -1,6 +1,5 @@
 const { pathsToModuleNameMapper } = require('ts-jest');
 const { compilerOptions } = require('./tsconfig.json');
-const { crypto } = require("crypto");
```

Comment:
not sure why this was added but it was stopping me from running one test (like passing a path to file and running only the tests in that file)